examples/mobile/HomeApp/index.html

```diff
@@ -43,6 +43,32 @@
 					!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='https://weatherwidget.io/js/widget.min.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','weatherwidget-io-js');
 				</script>
 			</div>
+
+			<div class="ui-content-subheader">
+				TV remote Control
```

Comment:
We discussed with @singa2000 that this widget should replace Weather widget.

Reply:
no no~ it's not need in this sample.
If I needed, I'll replace it.
Keeping the weather widget and keeping the TV remote contol widet, please?

Reply:
ok, sorry for misunderstanding